aep/general/0231/aep.md.j2

```diff
@@ -1,5 +1,153 @@
 # Batch methods: Get
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/42) for more
-information.
+Some APIs need to allow users to get a specific set of resources at a
+consistent time point (e.g. using a read transaction). A batch get method
+provides this functionality.
+
+## Guidance
+
+APIs **may** support batch get to retrieve a consistent set of resources.
+
+- The method's name **must** begin with `BatchGet`. The remainder of the method
+  name **should** be the plural form of the resource being retrieved.
```

Comment:
```suggestion
  name **must** be the plural form of the resource being retrieved.
```

Or is there a reason to keep this a should?

Reply:
"should" comes from the Google AIP. I presume that Google had reasons to make it "should", but I don't have that context. So I would ask the opposite -- are we sure it should be a "must"?